.pre-commit-config.yaml

```
@@ -0,0 +1,178 @@
+        name: "commit-msg: Create session commit message"
+        entry: scripts/prepare-commit-msg/create_session_commit.py
+        language: python
+        stages: [commit-msg]
+      - id: pytest-fast
+        name: "python: Run fast tests"
+        entry: pytest
+        language: python
+        types: [python]
+        args: ["-m", "not slow"] # Assumes you mark slow tests
+        pass_filenames: false
+        always_run: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)